Readme.md

```diff
@@ -1,26 +1,47 @@
 Slack app for Voting on Stories and Tickets remotely and asynchronously.
 
+## Usage
+
+1. Install this app to your Slack team visiting http://asyncvoter.tk and pressing on the `Add to Slack` button.
```

Comment:
@arun1595 As a developer/contributor Is up to you creating a new or using an existing team for testing purposes.